### PR TITLE
Clean up regex

### PR DESCRIPTION
**Pull request by 50Wliu**
:art: Clean up some really messy regex :art:

Before: `s(h(ift|ow(Mod(elessDialog|alDialog)|Help)))`
After: `shift|showModelessDialog|showModalDialog|showHelp`
:tada:

I probably missed something or messed up a capture group though, so :eyes: would be really appreciated!

### CODE REVIEW

**Comment by kevinsawicki** [comment]
This looks good at first glance, although it no longer merges cleanly since #154 was merged.

**Comment by 50Wliu** [comment]
I'm actually going to clean this up a bit more (some things could probably be regrouped like marginTop, marginRight, margin...).

**Comment by 50Wliu** [comment]
Fixed the merge conflicts.

Also went ahead and re-combined some things (only if there were a bunch like setTime, etc) and fixed a few bugs I found (eg `setChidl`).

File: grammars/javascript.cson [hunk 1]
```diff
@@ -419,19 +419,19 @@
...
+    'match': '(?<=\\.)(shape|systemId|scheme|scope|scrolling|standby|start|size|summary|specified|sectionRowIndex|selected|selectedIndex|hspace|httpEquiv|htmlFor|height|headers|href|hreflang|noResize|notations|notationName|noShade|noHref|nodeName|nodeType|nodeValue|noWrap|nextSibling|name|ch|childNodes|chOff|checked|charset|cite|content|cookie|cords|code|codeBase|codeType|cols|colSpan|color|compact|cells|cellSpacing|cellPadding|clear|className|caption|type|tBodies|title|tHead|text|target|tagName|tFoot|isMap|index|id|implementation|images|options|ownerDocument|object|disabled|dir|doctype|documentElement|docmain|declare|defer|defaultSelected|defaultChecked|defaultValue|dateTime|data|useMap|publicId|parentNode|profile|profileend|prompt|previousDibling|enctype|entities|event|elements|vspace|version|value|valueType|vLink|vAlign|URL|firstChild|form|forms|face|frame|frameBorder|width|link|links|longDesc|lowSrc|lastChild|lang|label|anchors|accessKey|accept|acceptCharset|action|attributes|applets|alt|align|archive|areas|axis|aLink|abbr|rows|rowSpan|rowIndex|rules|rev|referrer|rel|readOnly|multiple|method|media|marginHeight|marginWidth|maxLength|body|border|background|bgColor)\\b'
```
Comment (mnquintana) [inline grammars/javascript.cson]: Huh – this is a bit of an aside, but I just noticed that this line isn't syntax highlighted for some reason.

Comment (50Wliu) [inline grammars/javascript.cson]: Long lines on Github share the same limitations as on Atom (although I think the char limit is much longer on Github).

**Comment by kevinsawicki** [comment]
Thanks so much for cleaning this up, it will be much easier to maintain going forward :blue_heart:

**Pull request merged**